src/rpc/blockchain.cpp

```diff
@@ -1056,48 +1056,120 @@ bool FillScRecordFromInfo(const uint256& scId, const CSidechain& info, CSidechai
     if (bOnlyAlive && (scState != CSidechain::State::ALIVE))
     	return false;
 
-    int currentEpoch = (scState == CSidechain::State::ALIVE)?
-            info.EpochFor(chainActive.Height()):
-            info.EpochFor(info.GetCeasingHeight());
-
     sc.push_back(Pair("scid", scId.GetHex()));
-    sc.push_back(Pair("balance", ValueFromAmount(info.balance)));
-    sc.push_back(Pair("epoch", currentEpoch));
-    sc.push_back(Pair("end epoch height", info.StartHeightForEpoch(currentEpoch +1) - 1));
-    sc.push_back(Pair("state", CSidechain::stateToString(scState)));
-    sc.push_back(Pair("ceasing height", info.GetCeasingHeight()));
-
-    if (bVerbose)
+    if (!info.IsNull() )
```

Comment:
If there are unconfirmed fwts in mempool, this `info` would contain their amount at height `-1`. I believe these amounts may appear twice, in `maturityHeight` and `unconf maturityHeight`.
I believe we should keep `unconf maturityHeight` only and change `GetSidechain `in txmempool